resources/public/api/conf/P1/application.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 1.0
+---
+
+title: Individuals
+version: 1.0
```

Comment:
Should the version here be `P1`?

Reply:
discussed this with @tom59 yesterday, and we think this is the actual _raml_ file version